app/src/main/java/com/andela/art/securitydashboard/SecurityDashboardActivity.java

```diff
@@ -41,4 +44,25 @@ public boolean onOptionsItemSelected(MenuItem item) {
         startActivity(settings);
         return true;
     }
+
+    @Override
+    public void onBackPressed() {
+        if (backButtonToExitPressedTwice) {
+            super.onBackPressed();
+            finish();
+            moveTaskToBack(true);
+        }
+
+        this.backButtonToExitPressedTwice = true;
+        Toast.makeText(this, "Press BACK BUTTON again to exit.",
```

Comment:
Change the Toast message wordings to `Press again to exit.`

Reply:
Done.